js/header.js

```diff
@@ -0,0 +1,21 @@
+window.addEventListener('scroll', function() {
+  let scroll = pageYOffset;
+  let width = screen.width;
+if (scroll > 150 && width > 880) {
```

Comment:
add comments what are these magic numbers. why you choose them.
for example 150 its header height and so on